heron/common/tests/cpp/network/switch_unittest.cpp

```diff
@@ -68,16 +67,19 @@ class Terminate : public Client {
 
 static TestServer* server_;
 
-void start_server(sp_uint32 port) {
+void start_server(sp_uint32* port, CountDownLatch* latch) {
```

Comment:
This function is run as thread with no return value.
I tried reference, but compiler complained. I suggest pointer.